mycore-ifs/src/main/java/org/mycore/datamodel/niofs/ifs2/MCRFileSystemUtils.java

```diff
@@ -309,4 +316,30 @@ private static Optional<MCRDirectory> doResolveParent(MCRPath parent) {
             .map(MCRDirectory.class::cast);
     }
 
+    static Collection<String> getObjectBaseIds() throws IOException {
+        final Path baseDir = Paths.get(getBaseDir());
+        if (!Files.isDirectory(baseDir)) {
+            return List.of();
+        }
+        try {
+            return Files.list(baseDir)
```

Comment:
The Stream should be closed, i think that can cause issues on windows otherwise.

Reply:
`The returned stream encapsulates a [DirectoryStream](https://docs.oracle.com/javase/8/docs/api/java/nio/file/DirectoryStream.html). If timely disposal of file system resources is required, the try-with-resources construct should be used to ensure that the stream's [close](https://docs.oracle.com/javase/8/docs/api/java/util/stream/BaseStream.html#close--) method is invoked after the stream operations are completed.`

Reply:
OK. Good point!